go.mod

```diff
@@ -10,6 +10,7 @@ require (
 	github.com/gorilla/websocket v1.4.2
 	github.com/pkg/errors v0.9.1
 	github.com/status-im/keycard-go v0.3.3
+	github.com/tyler-smith/go-bip39 v1.0.1-0.20181017060643-dbb3b84ba2ef
```

Comment:
@bitgamma do you think it's fine to use this dependency?
It was used as the reference implementation for status-go:
https://github.com/status-im/status-go/blob/a82de9f5f916f8b4118dfc15bcae81b2487d673a/extkeys/mnemonic.go#L27

Reply:
looks fine to me, seems like it being maintained